csrc/preseg_passes/allocation_order_inference.cpp

```diff
@@ -170,8 +181,21 @@ TensorView* AllocationOrderInferencer::resolveAllocationOrder(
       continue;
     }
 
+    auto iter = alloc_order_map_.find(tv);
     // skip entry that doesn't have an allocation order
```

Comment:
This comment is no longer consistent with the new code -- we return null instead of "skipping" the entry.

Reply:
Btw, are you sure this change of behavior is fine?

Reply:
Good catch. We are just stopping the propagation. I'll update the comment.

> are you sure this change of behavior is fine?
I'd prefer to have propagation failure, which motivates us to put in more propagation rule. Rather than having a propagation suggesting an allocation order on output that causes a perf regression. :crossed_fingers: